iotlabcli/parser/experiment.py

```diff
@@ -700,13 +701,14 @@ def get_experiment_parser(opts):
         # Add a 'date' field
         timestamp = ret['start_time']
         ret['local_date'] = time.ctime(timestamp) if timestamp else 'Unknown'
-        return ret
     elif opts.get_cmd == 'experiments':
-        return experiment.get_active_experiments(api,
-                                                 running_only=not opts.active)
+        ret = experiment.get_active_experiments(api,
+                                                running_only=not opts.active)
     else:
         exp_id = helpers.get_current_experiment(api, opts.experiment_id)
-        return experiment.get_experiment(api, exp_id, opts.get_cmd)
```

Comment:
I would rather locally disable 'no-else-return` here.  With `return` in every branch I know its done and no other handling will be required later.